### ISSUE TITLE
Azan word in counter

### ISSUE THREAD

**Issue by MShoebPT** [mawaqit/android-tv-app#1117]
Can add the word "azan" to the counter that appears in the announcement.
This counter ends when the azan sign appears and then another begins that says "iqamah in".
Masjid Nabawi screen sample.
jazakAllah!

"Assr azan in .."
"Fajr azan in ..."
Isha azan in .."

![IMG_1510](https://github.com/mawaqit/android-tv-app/assets/158289808/ea1992c7-a26c-4041-b45f-7c29b66b322a)

**Comment by ibrahim-zehhaf-mawaqit** [comment]
Why adding azan is important ?
When we say Fajr in xx:xx this means azan fajr in xx:xx with less text

**Comment by MShoebPT** [comment]
Yes, because as you know here in Portugal and I think in the UK too, we give more importance to Iqamah (Jamaat) and when people look at the counter they think that xx:xx is for Iqamah.
And since there is another counter for Iqamah after Azan I think it makes sense for this one to say "azan in")

**Comment by ibrahim-zehhaf-mawaqit** [comment]
Let's add it in EN and PT language
